sdk/tests/conformance/resources/gl-object-get-calls.js

```diff
@@ -122,6 +121,11 @@ gl.bindTexture(gl.TEXTURE_2D, null);
 var framebuffer = gl.createFramebuffer();
 gl.bindFramebuffer(gl.FRAMEBUFFER, framebuffer);
 gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, texture, 0);
+var colorAttachmentsNum = 1;
+if (contextVersion > 1) {
+  colorAttachmentsNum = gl.getParameter(gl.MAX_COLOR_ATTACHMENTS);
+  gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0 + colorAttachmentsNum - 1, gl.TEXTURE_2D, texture, 0);
```

Comment:
I don't understand the intent of this code -- here you're reattaching the texture to the attachment point COLOR_ATTACHMENT0 + MAX_COLOR_ATTACHMENTS - 1, but below you're verifying all of the attachment points between 0 and COLOR_ATTACHMENT0 + MAX_COLOR_ATTACHMENTS - 1, inclusive. Should this code here not attach the texture to all of those attachment points too?